fluentd/Gemfile

```diff
@@ -5,6 +5,7 @@ gem 'elasticsearch'
 gem 'fluent-plugin-kubernetes_metadata_filter'
 gem 'fluent-plugin-concat'
 gem 'fluent-plugin-elasticsearch'
+gem 'fluent-plugin-kafka'
```

Comment:
@alanconway do you think we should start locking down versions to be repeatable here and not be "surprised" by updates.

Reply:
@jcantrill For this purpose i've added 767c29f5b2f2f05982884da73d3be50dd25e98fa to expedite this PR.